sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/DocumentQueryExecutionContextFactory.java

```diff
@@ -127,7 +127,7 @@
                                                                                                               resourceTypeEnum,
                                                                                                               resourceType,
                                                                                                               query,
-                                                                                                              queryRequestOptions,
+                                         cosmosQueryRequestOptions,
```

Comment:
indentation.

Reply:
ah intellij, will fix it here and everywhere else.